src/components/app-shell.js

```diff
@@ -163,6 +163,7 @@ class AppShell extends localize(i18next)(connect(store)(LitElement)) {
             <a href="/aqua" ?selected=${this.page === 'aqua'}>Aqua</a>
           </nav>
           <nav class="second">
+            <a rel="noopener" href="https://nbviewer.jupyter.org/github/Qiskit/qiskit-tutorial/blob/master/index.ipynb">${i18next.t('tutorials')}</a>
```

Comment:
Makes sense to add a `target="_blank"` to not to lose the user, since is an external link?